src/scripts/Player.ts

```diff
@@ -18,6 +18,8 @@ class Player {
     private _shinyList: boolean[];
     private _notCaughtBallSelection: GameConstants.Pokeball;
     private _alreadyCaughtBallSelection: GameConstants.Pokeball;
+    private _sortOption: KnockoutObservable<string>;
```

Comment:
I was planning to have an array of objects like `{optionText: Level, optionValue: "levelObservable"}` since sometimes the option text should be different to the property name. Also I think this allows for returning something other than a string.

Is it possible to create an enum which would become an array like that?